XamarinCommunityToolkit.UnitTests/ConvertersNamespace_Tests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Linq;
+using System.Reflection;
+using Xamarin.CommunityToolkit.Behaviors;
+using Xamarin.Forms;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests
+{
+	public class Namespace_Tests
+	{
+		[Fact]
+		public void MakeSureConvertersAreInTheRightNamespace()
+		{
+			var allTheTypes = Assembly.GetAssembly(typeof(BaseBehavior)).GetTypes();
+			foreach (var type in allTheTypes.Where(t => t.Name.EndsWith("Converter") && t.GetInterface("IValueConverter") != null))
+			{
```

Comment:
Please don't use braces for one line loops.

Dura lex sed lex (c)

Reply:
Whoops. Done